remotedcurl/remotedcurl.c

```diff
@@ -0,0 +1,135 @@
+#include <assert.h>
+
+#include "amqp.h"
+#include "amqp_tcp_socket.h"
+
+#include "constants.h"
+#include "dcurl.h"
+
+#define HOSTNAME "localhost"
+#define PORT 5672
+#define HEATBEATS 0  // disables heartbeats
+#define ACCOUNT "guest"
```

Comment:
Don't hardcode configurations here.

Reply:
> Don't hardcode configurations here.

[process_all_options](https://github.com/alanxz/rabbitmq-c/blob/a65c64c0efd883f3e200bd8831ad3ca066ea523c/tools/common.c#L428) in RabbitMQ C client provides an implementation for options. Do we reuse it? Or, we choose an implementation for [simple .INI file parser](https://github.com/benhoyt/inih).

Reply:
`dcurl` is used in the form of library, and it would be suitable to configure it via default options and/or specific configuration file.

Reply:
We maybe require tow configuration files, one is for remote interface expected as the link https://github.com/DLTcollab/dcurl/issues/91#issuecomment-469523999 in client sides and the other is for dcurl workers.

And, Can I reuse [simple .INI file parser](https://github.com/benhoyt/inih) for configuration files?

Reply:
> `dcurl` is used in the form of library, and it would be suitable to configure it via default options and/or specific configuration file.

We maybe require two configuration files, one is for remote interface expected as the link https://github.com/DLTcollab/dcurl/issues/91#issuecomment-469523999 in client sides and the other is for dcurl workers.

And, Can I reuse simple .INI file parser for configuration files?

Reply:
Don't depend on configure file manipulation library at first glance. Think about the scenario of IRI + dcurl + remote interface where it is not suitable to specify extra files for configurations. Instead, we have to follow Java-style assignment. Take the above into consideration, we shall eliminate the numbers of configuration entries as possible.

Reply:
At the moment, only RMQ_HOST is required for configuration purpose. Something like `localhost:5672` might fit `RMQ_HOST`.

Reply:
In scenario of IRI + dcurl + remote interface, remote interface is composed of client, RabbitMQ broker and dcurl worker. As above discusses, we only requires RabbitMQ host IP setting and we use default values for other configurations in client side. On the other hand, dcurl worker also does this. In addition, for convenient setting of dcurl worker, I plan to use command line setting. 
```
./remotedcurl localhost 5672
```

Reply:
The runtime parameter `localhost:5672` is expected to appear as an entry of IRI configurations. Keeping it simple and stupid is always a good policy.